Remove duplicate results from Albums

RaihanAli29 opened the pull request:
Added a filter + findIndex function to remove duplicate album names upon display.

RaihanAli29 committed:
Remove duplicate results from Albums

Added a filter + findIndex function to remove duplicate album names upon display.

volfovaklara commented on assets/js/script.js:
```diff
@@ -8,17 +8,16 @@ function setCurrentDay() {
 setCurrentDay();
 
 // function to display selected dropdown item using user input
-$(document).ready(function() {
-    $(".dropdown-item").click(function() {
-      var selectedGenre = $(this).text();
-      $("#dropdownInput").val(selectedGenre);
-    });
```
This could be deleted. It's the same code tbf.

volfovaklara commented on assets/js/script.js:
```diff
@@ -8,17 +8,16 @@ function setCurrentDay() {
   });
+});
 
-// Function to get a random artist based on the selected genre and era
```
Delete it, just differently worder comment.

volfovaklara commented on assets/js/script.js:
```diff
@@ -8,17 +8,16 @@ function setCurrentDay() {
+// Function to fetch and display a random artist
 function getRandomArtist() {
-  // Get the selected genre from the dropdown
-  let selectedGenre = $('#dropdownInput').val();
```
Same here, just different variable, I suppose the function of it is the same.

RaihanAli29 committed:
Set function to filter out duplicate album results

‘Set’ function uses a Set to keep track of unique album names. It iterates through the array of albums, converts each album title to lowercase for case-insensitive comparison, and checks whether the album name is already in the set. If not, it adds the album name to the set and includes the album in the filtered array.
